### ISSUE TITLE
Squid plots just for fishery data?

### ISSUE THREAD

**Issue by andrew-edwards** [cgrandin/hake-assessment#410]
So we had a Canadian conference call yesterday and Shannon asked about having the squid plots for just fishery (or maybe it was survey) data. We tried to explain that this isn't possible since it's model output that uses all the data, but she said that  she thought that Ian had said (last year) that we could look into it. I'm guessing there's some misunderstanding somewhere - any ideas Ian?

**Comment by taylori** [comment]
I don't remember what I said.

It would definitely be possible (and probably informative) to run a sensitivity analysis where we turn off all survey age comps (negative fleet number would turn off their contribution to the likelihood while still reporting the implied fit). Or turn off both survey age comps and index as well. And then we could do retrospectives and make a squid plot from it. I'll go ahead and try this in case it's useful, but I don't think we need to commit to providing anything at this point.

**Comment by andrew-edwards** [comment]
I'm not sure if it's worth bothering for now. Also, we only report MLEs in the squid plots, and presumably the uncertainty is much higher when leaving out one set of data, so the plots may even turn out a little misleading. Maybe...

**Comment by taylori** [comment]
Having thought about this a bit more, I thought of a reasonable pair of sensitivities that I will put together and keep available in case this gets brought up at the SRG. The two sensitivities would include one where the fishery ages are given more weight relative to the survey ages and the opposite, where the survey ages are given little weight. 

These models could then have retrospectives run to see the impact on the squid plot. I'm guessing that a model with reduced weight on fishery ages would take longer to settle in on the relative strength of each cohort, but you would also avoid the initial overestimates of some cohorts, like the 2014 year class.

Taking out the ages entirely instead of just reducing the weighting would require fixing the selectivity parameters, which I think would make the retrospectives less realistic.

**Comment by aaronmberger** [comment]
I like that plan and second it.  It will be quite interesting to see the squid plot from each.

**Comment by taylori** [comment]
Here are additional squid plots for models with either the fishery or survey downweighted to 5% of the input sample size (as opposed to base model values of about 45% for the fishery and 92% for the survey). I allowed the theta parameters for the non-downweighted fleet to remain estimated and there was little change from the base value.

Here are a few conclusions as I see them:
- Downweighting the fishery ages results in slower determination of the size of each cohort (estimates at age 2 are closer to 0 than base)
- Downweighting the fishery ages results also results in shrinkage of final estimates toward 0
- Downweighting the survey ages results in more up and down waves, perhaps because the survey with selectivity constant across years provides some stability that's missing when those ages are downweighted.

I'm not sure if it's worth including these in a presentation or document (although it might make Shannon happy), but in case they seem useful, I uploaded .eps versions of these figures to main-figures in commit edb45489d7bdb7bf0e1dcf82572e9beed491881e. If not, we can easily delete. The code used to make these is a bit sloppy and not in function format. Would it be worth creating a new folder in doc/r/ with a name like "scripts" or "notes" to store things like this that might be useful to sharing but aren't generalized or in function format.

![squid_plot_base](https://user-images.githubusercontent.com/4992918/36562972-8f901e98-17cd-11e8-9acd-ba9ec5f548b3.png)
![squid_plot_downweight_fishery_ages](https://user-images.githubusercontent.com/4992918/36562974-8fb08f20-17cd-11e8-843e-f26e6c58ce5e.png)
![squid_plot_downweight_survey_ages](https://user-images.githubusercontent.com/4992918/36562975-8fd10c5a-17cd-11e8-9daf-337068f76618.png)
